src/read_next/read_next.css

```
@@ -1,3 +1,26 @@
+.pagination_data_post {
+    display: flex;
+    gap: 20px;
+}
+
+.pagenation_post {
```

```suggestion
.pagination_post {
```
correct spelling is important